crates/bevy_render/src/view/mod.rs

```diff
@@ -160,13 +160,13 @@ impl Default for ColorGrading {
 
 #[derive(Clone, ShaderType)]
 pub struct ViewUniform {
-    view_proj: Mat4,
-    unjittered_view_proj: Mat4,
-    inverse_view_proj: Mat4,
-    view: Mat4,
-    inverse_view: Mat4,
-    projection: Mat4,
-    inverse_projection: Mat4,
+    world_to_clip: Mat4,
+    unjittered_world_to_clip: Mat4,
+    clip_to_world: Mat4,
+    view_to_world: Mat4,
+    world_to_view: Mat4,
+    view_to_clip: Mat4,
+    clip_to_view: Mat4,
```

Comment:
We should probably have comments here and in wgsl labeling the matrices.

// Camera projection * inverse view
world_to_clip: Mat4,